HaikuPorter/BuildMaster.py

```diff
@@ -813,12 +188,6 @@ def __init__(self, portsTreePath, packagesPath, options):
 
 		self.localBuilders = getOption('localBuilders')
 		self.remoteAvailable = False
-		if paramiko:
-			self.remoteAvailable = True
-		else:
-			print('Remote mode unavailable')
```

Comment:
I dropped this logic because it seemed unnecessary.  Local Builders appear to be for running buildmaster natively on Haiku building locally.

localBuilder here is *not* running ./haikuporter on haiku.  This is purely --build-master logic.

The remoteAvailable = true moved down to after successfully adding a remote builder.